src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesUpgrade.Codeunit.al

```diff
@@ -0,0 +1,21 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+codeunit 328 "No. Series Upgrade"
+{
+    Access = Internal;
+    Subtype = Upgrade;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    trigger OnUpgradePerCompany()
+    var
+        NoSeriesInstaller: Codeunit "No. Series Installer";
+    begin
+        NoSeriesInstaller.SetupNoSeriesImplementation();
```

Comment:
Please make sure that the new upgrade tag is registered to the subscribers so we insert it with the new deployments.